.pullapprove.yml

```diff
@@ -39,3 +39,17 @@ groups:
       enabled: true
       approve_regex: '^qa-passed'
       reject_regex: '^qa-failed'
+
+  docs-team:
+    conditions:
+      files:
+        - "docs/*"
+        - "*.md"
+        - "*.rst"
+    required: 1
```

Comment:
Yes Mike Tullis could can provide coverage. I'll get his Github username for you and reply here.